Mds/new mobile sdk/okta 503880 revise quickstart (#3424)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/configmid.md

> The SDK loads the configuration values from a configuration file named `Okta.plist` if it exists in your main bundle. You may also specify the values in code using `WebAuthentication(issuer:clientId:scopes:responseType:redirectUri:logoutRedirectUri:additionalParameters:)`.

Reviewer: ```suggestion
The SDK loads the configuration values from a configuration file named `Okta.plist` if it exists in your main bundle. You may also specify the values in code using `WebAuthentication(issuer:clientId:scopes:redirectUri:logoutRedirectUri:additionalParameters:)`.
```

Recent updates dropped the `responseType:` argument.

Author: Updated in next commit

> The iOS SDK uses information from the application integration you created earlier to connect to the Okta org. The SDK loads these values from a configuration file named `Okta.plist` if it exists in your main bundle. You may also specify the values in code using `WebAuthentication(issuer:clientId:scopes:responseType:redirectUri:logoutRedirectUri:additionalParameters:)`.

Author: Nice catch. It will be in the next commit.

> The iOS SDK uses information from the application integration you created earlier to connect to the Okta org. The SDK loads these values from a configuration file named `Okta.plist` if it exists in your main bundle. You may also specify the values in code using `WebAuthentication(issuer:clientId:scopes:responseType:redirectUri:logoutRedirectUri:additionalParameters:)`.

Author: The dangers of stack selectors... especially when you didn't write the main content. I removed the first sentence and and modified the second:
"The SDK loads the configuration values from..."

So far I'm planning on putting that in the next commit. Let me know if it should be different.

> The iOS SDK uses information from the application integration you created earlier to connect to the Okta org. The SDK loads these values from a configuration file named `Okta.plist` if it exists in your main bundle. You may also specify the values in code using `WebAuthentication(issuer:clientId:scopes:responseType:redirectUri:logoutRedirectUri:additionalParameters:)`.

Reviewer 2: No thats fine

> Create the Okta configuration file and add the keys and values for your application integration:

Author: Not really. by default it will be created in the active directory, which is the top level of the project contents. However, all plist files are moved into the default bundle during compilation unless they are specifically assigned to a different bundle.

> Create the Okta configuration file and add the keys and values for your application integration:

Reviewer 2: ok - then never mind.

   | --- | ----- |
   | `clientId` | The client ID from the app integration that you created, such as `0ux3rutxocxFX9xyz3t9` |
   | `issuer` | The domain of your registered Okta org followed by `/oauth2/default`, such as `https://dev-1234567.okta.com/oauth2/default` |
   | `logoutRedirectUri` | The sign-out redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/` |

>    | `redirectUri` | The Sign-in redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/callback` |

Reviewer 2: Sorry but you had sign-out in line above and Sign-in here. Couldn't take it.....

```suggestion
   | `redirectUri` | The sign-in redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/callback` |
```

>    | `logoutRedirectUri` | The post-logout redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/` |

Reviewer 2: Easier to give it the name in the Admin Console?

```suggestion
   | `logoutRedirectUri` | The sign-out redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/` |
```

Author: Done. In the next commit.

>    | `redirectUri` | The Redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/callback` |

Reviewer 2: Easier to give it the name in the Admin Console?

```suggestion
   | `redirectUri` | The Sign-in redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/callback` |
```

Author: Done. In the next commit.
